fiddler/requirements-dev.txt

```
@@ -0,0 +1 @@
+datadog-checks-dev
```

```suggestion
datadog-checks-dev
fiddler-client==1.4.0
```

Add the fiddler-client package to the requirements-dev.txt file so that the E2E tox environment will install the package during E2E development. Note, I upgraded the version to 1.4.0 and no longer saw the numpy import errors.